### ISSUE TITLE
[BUG] WebSocket is closed before the connection is established

### ISSUE THREAD

**Issue by 404alya** [soketi/pws#1231]
**Description**
When I try to connect, I get error "WebSocket connection to 'wss://ws.example.com' failed: WebSocket is closed before the connection is established.". After waiting for a while or sometimes after turning it off and on, it gets fixed. The problem seems to be in the client code, but it's actually not. I tried connecting to the server with react, html and tested it on some websocket test sites. I have the same problem on both localhost and hetzner servers regardless of TLS. Server logs only show that the user's connection was closed and there is no reason

**Reproduction steps**
1. Start server
2. Try to connect

**Expected behavior**
should work properly

**Environment**
- Soketi version: All versions 
- Adapter: local
- App Manager: array
- Queue: sync
- Cache Managers: memory

**Comment by bbeng89** [comment]
Did you ever find a solution to this? I am experiencing the same thing. Stopping the server and starting it again fixes it temporarily, but the problem eventually comes back. It seems I'm hitting a limit somewhere but I can't find it. Checked file descriptor limits, nginx limits, etc. but haven't found a fix yet.

**Comment by 404alya** [comment]
> Did you ever find a solution to this? I am experiencing the same thing. Stopping the server and starting it again fixes it temporarily, but the problem eventually comes back. It seems I'm hitting a limit somewhere but I can't find it. Checked file descriptor limits, nginx limits, etc. but haven't found a fix yet.


No, I didn't. I just switched to [socket.io](https://socket.io/).
